src/_util.js

```diff
@@ -1,5 +1,41 @@
 'use strict'
 
+/**
+ * Inherit the prototype methods from one constructor into another.
+ * @param {function} ctor Constructor function which needs to inherit the prototype.
+ * @param {function} superCtor Constructor function to inherit prototype from.
+ * @private
+ */
+function inherits(ctor, superCtor) {
```

Comment:
Was this polyfill copied from somewhere? If so, could you add a link to it in the function docs?

Reply:
Added link to polyfill repo